tests/unit/matrix/test_subtotals.py

```diff
@@ -200,6 +201,102 @@ def it_can_compute_a_subtotal_row_to_help(self, request):
         )
 
 
+class DescribePairwiseSigTestSubtotals(object):
+    """Unit test suite for `cr.cube.matrix.PairwiseSigTestSubtotals` object."""
+
+    def it_provides_a_blocks_interface_method(self, request, _init_):
+        property_mock(
+            request,
+            PairwiseSigTestSubtotals,
+            "_blocks",
+            return_value=np.array([[[1], [2]], [[3], [4]]]),
+        )
+
+        blocks = PairwiseSigTestSubtotals.blocks(None, None, None, None, 0)
+
+        _init_.assert_called_once_with(ANY, None, None, None, None, 0, True, True)
+        assert blocks.tolist() == [[[1], [2]], [[3], [4]]]
+
+    def it_can_compute_a_intersection_cell_value_to_help(self):
+        assert np.isnan(
+            PairwiseSigTestSubtotals(None, None, None, None, 0)._intersection(
+                None, None
+            )
+        )
+
+    @pytest.mark.parametrize(
+        ("addend_idxs", "subtrahend_idxs", "diff_cols_nan", "expected_value"),
```

Comment:
I don't really understand the `diff_cols_nan` and what it's supposed to mean... But this is just at the first glance, I might be able to understand it more after thorough review.

Reply:
Also, expected value for what? I'm not clear on how the `_subtotal_column` is supposed to work. Its API is a bit weird, where you have to pass it the subtotal and a column... I hope I'll understand it better by the time I finish with the review.

Reply:
Basically when the `diff_cols_nan` is True all columns and rows subtotals will be nan. This means that for `PairwiseSigTestSubtotals` if the param is True all the cols or row have to be full of np.nan that are dashes in the exporter. So for each column i need to pass the subtotal and the column itself cause we have the cases that we have to return the col itself of nan if diff_cols_nan is true and if the subtotal has subtraends